aptos-move/aptos-vm/src/block_executor/vm_wrapper.rs

```diff
@@ -138,8 +139,12 @@ impl<'a, S: 'a + StateView + Sync> ExecutorTask for AptosExecutorTask<'a, S> {
             None => MoveStorageOp::Delete,
         };
 
+        debug_assert!(
+            matches!(key.inner(), StateKeyInner::AccessPath(ap) if ap.is_resource_group()),
+            ""
```

Comment:
Let's do this earlier in the method? and also add a message?
"Should only be used to convert resource group blobs to WriteOp" or smt

Reply:
Ooops, converted to assert and forgot to put the message inside🫠